src/seq/mod.rs

```diff
@@ -451,6 +451,18 @@ impl<'a, S: Index<usize, Output = T> + ?Sized + 'a, T: 'a> ExactSizeIterator
 }
 
 
+// Sample a number uniformly between 0 and `ubound`. Uses 32-bit sampling where
+// possible, primarily in order to produce the same output on 32-bit and 64-bit
+// platforms.
```

Comment:
Maybe add `#[inline]` to encourage LLVM?

Reply:
Makes sense but has negligible effect on benchmarks (`seq_iter_choose_from_1000` and `seq_iter_window_hinted_choose_from_1000` still being about 12% slower than before this PR). But we can live with this small hit.